src/modules/product/ProductTypes.ts

```diff
@@ -10,56 +10,82 @@ export interface Product {
   payments?: nullable<ProductPayment[]>
   gtin?: string
   mpn?: string
-  additional_shipping_time?: number
-  external_id?: string
-  category_default_id?: number
-  hotsite_id?: number
+  additionalShippingTime?: number
+  externalId?: string
+  categoryDefaultId?: number
+  hotsiteId?: number
   description?: string
-  short_description?: string
+  shortDescription?: string
   relevance?: number
   tags?: string
-  min_quantity?: number
-  max_quantity?: number
-  sell_in_kit_only?: nullable<boolean>
-  meta_title?: string
-  meta_description?: string
-  meta_keywords?: string
-  kit?: nullable<boolean>
-  kit_markup?: number
-  is_virtual?: nullable<boolean>
-  is_pre_sale?: nullable<boolean>
+  minQuantity?: number
+  maxQuantity?: number
+  isSellInKitOnly?: boolean
+  metaTitle?: string
+  metaDescription?: string
+  metaKeywords?: string
+  isKit?: boolean
+  kitmarkup?: number
+  isVirtual?: boolean
+  isPreSale?: boolean
   video?: string
   images?: ProductImage[]
   weight?: number
   depth?: number
   width?: number
   height?: number
-  sell_out_of_stock?: nullable<boolean>
-  additional_time_out_of_stock?: number
+  isSellOutOfStock?: boolean
+  additionalTimeOutOfStock?: number
   balance?: number
   price?: number
-  min_price_range?: number
-  max_price_range?: number
-  has_price_range?: nullable<boolean>
-  price_compare?: number
+  minPriceRange?: number
+  maxPriceRange?: number
+  isHasPriceRange?: boolean
```

Comment:
É esse mesmo o nome, `isHasPriceHange`? E não `hasPriceHange`

Reply:
@antonydosreis tu estava correto, não existe aquele "is". Ajustado.

Reply:
Tem que ver depois se tem que ser ajustada alguma nonenclatura no projeto do tema